<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpotifyConfigWidget</class>
 <widget class="QWidget" name="SpotifyConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lblUsername">
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineUsername"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblPassword">
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="linePassword">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnLogin">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Note: &lt;br/&gt;Spotify service is only avaiable in certain countries. And content and policy may vary in different regions.&lt;/p&gt;&lt;p&gt;Only Spotify Premium user can use this feature. Spotify Premium subscription can be purchased on &lt;a href=&quot;http://www.spotify.com/get-spotify/premium/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#508ed8;&quot;&gt;Spotify's official website&lt;/span&gt;&lt;/a&gt;.&lt;/p&gt;&lt;p&gt;The username is the id displayed in your Spotify profile page. If you are logged in by using Facebook, your displayed username may be different from your login id of Facebook, and the password is the same as your Facebook account password.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
